TaskLayer/GPTMDTask/GPTMDParameters.cs

```diff
@@ -8,14 +8,14 @@ public class GptmdParameters
         public GptmdParameters()
         {
             ListOfModsGptmd = GlobalVariables.AllModsKnown.Where(b =>
-                b.modificationType.Equals("N-linked glycosylation") ||
-                b.modificationType.Equals("Other glycosylation") ||
-                b.modificationType.Equals("Mod") ||
-                //b.modificationType.Equals("Artifact") ||
-                //b.modificationType.Equals("Biological") ||
-                b.modificationType.Equals("PeptideTermMod") ||
-                b.modificationType.Equals("Metal") ||
-                b.modificationType.Equals("ProteinTermMod")).Select(b => (b.modificationType, b.id)).ToList();
+                b.ModificationType.Equals("N-linked glycosylation") ||
```

Comment:
guess we need to decide what we're going to offer and what is available in the lists